src/__init__.py

```diff
@@ -6797,7 +6796,10 @@ def uri_to_dict(uri):
                     if ftab[1].startswith("page="):
                         self.kind = LINK_GOTOR
                         self.file_spec = ftab[0]
-                        self.page = int(ftab[1][5:]) - 1
+                        self.page = int(ftab[1].split("&")[0][5:]) - 1
+            elif ":" in self.uri:
```

Comment:
Could we tighten this to look for a colon only after an initial set of alphabetical characters?

For example:
```
if re.match('^[a-z]+:', self.uri)`:
    ...
```